platform_versioned_docs/version-23.4/enterprise/configuration/overview.mdx

```diff
@@ -495,14 +495,21 @@ Do not replace the [Seqera-provided default image](../../functionality_matrix/fu
 
 ## Seqera API
 
-Enable the API endpoints to host the Seqera Enterprise OpenAPI specification and use the [tw CLI](https://github.com/seqeralabs/tower-cli).
+Enable the API endpoints to host the Seqera Enterprise OpenAPI specification and use the [tw CLI](https://github.com/seqeralabs/tower-cli). Set custom API rate limits and timeouts. 
+
+:::note
+To configure API rate limit environment variables, you must add `ratelim` to the `MICRONAUT_ENVIRONMENTS`. Without `ratelim` being set, the rate limit configuration variables below are ignored.
+::: 
 
 <Tabs>
 <TabItem value="Environment variables" label="Environment variables" default>
 
 | Environment variable   | Description                                                                   | Value           |
 | ---------------------- | ----------------------------------------------------------------------------- | --------------- |
 | `TOWER_ENABLE_OPENAPI` | Enable the OpenAPI documentation endpoint, e.g., [cloud.seqera.io/openapi/index.html](https://cloud.seqera.io/openapi/index.html). | Default: `true` |
+| `TOWER_RATELIMIT_PERIOD` | Specify the number of HTTP requests that can be made during the `TOWER_RATELIMIT_REFRESH` period. | Default: `20` |
+| `TOWER_RATELIMIT_REFRESH` | API rate limit refresh period. | Default: `1s` |
+| `TOWER_RATELIMIT_TIMEOUT` | The waiting period before resending requests that exceed the allowed limit defined in `TOWER_RATELIMIT_PERIOD` during the refresh period. | Default: `500ms` |
```

Comment:
This is the waiting period before **rejecting** requests. In case the period is refreshed before reaching the timout, the request could get to be processed. The following diagrams may clarify the behavior:

Supposing:
- `TOWER_RATELIMIT_PERIOD`: `2`
- `TOWER_RATELIMIT_REFRESH`: `400ms`
- `TOWER_RATELIMIT_TIMEOUT`: `300ms`

Timeout expired:
![image](https://github.com/user-attachments/assets/91207075-e66a-4400-8cb6-ac4db59d6fcd)

Enqueued request processed in next period:
![image](https://github.com/user-attachments/assets/d66ac185-572c-408a-816f-f92a86bedd46)